### ISSUE TITLE
Inline markdown links are broken (in both senses)

### ISSUE THREAD

**Issue by detly** [randy3k/AutoWrap#21]
If I have an inline markdown link such as `[link text](https://github.com)`, and it's near the end of a line, I can end up with:

```
[link text]
(https://github.com)
```

This won't render as a link in most markdown renderers. I don't want to simply drop `(` from the list of breaking characters, but I don't really know how you'd solve this without having per-syntax exceptions.

(This could be a bug in Wrap Plus too, which I'm also using. See their [issue #57](https://github.com/ehuss/Sublime-Wrap-Plus/issues/57).)

**Comment by detly** [comment]
Oh, I don't think it's Wrap Plus, because **they** break it between `link` and `text` if I trigger it manually.

**Comment by randy3k** [comment]
You could edit the setting `auto_wrap_break_patterns`.

**Comment by detly** [comment]
I thought about that, but what would I do other than remove `(`? Maybe putting that in markdown specific settings would work, and any other occurrences of `(` would likely be preceded by a space.

**Comment by randy3k** [comment]
Yes, you would put the following in markdown specific settings `Markdown.sublime-settings`.

```json
{
    "auto_wrap_break_patterns" :  ["\\[", " \\(", "\\{", " ", "\\n"]
}
```
